getting-started/php/devfile.yaml

```
@@ -0,0 +1,51 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: php-web-simple
+attributes:
+  persistVolumes: 'false'
+projects:
+-
+    name: php-web-simple
+    source:
+        type: git
+        location: "https://github.com/che-samples/web-php-simple"
+components:
+-
+  type: chePlugin
+  id: redhat/php/latest
+  memoryLimit: 1Gi
+-
+  type: chePlugin
+  id: redhat/php-debugger/latest
+  memoryLimit: 256Mi
+-
+  type: dockerimage
+  alias: php
+  image: quay.io/eclipse/che-php-7:nightly
+  memoryLimit: 512Mi
```

I've just merged github.com/eclipse/che-devfile-registry/pull/67, we should include the endpoint fix from there.
```suggestion
  memoryLimit: 512Mi
  endpoints:
    - name: '8080/tcp'
      port: 8080
```